src/components/EndorsementChain/EndorsementChainLayout/EndorsementChainLayout.tsx

```diff
@@ -193,7 +171,7 @@ const DetailsEntity: React.FunctionComponent<DetailsEntityProps> = ({ title, add
       <div className="flex flex-nowrap pr-8">
         <div className="relative flex-shrink-0 lg:hidden" style={{ width: "40px" }}>
           <div className="absolute left-0 right-0 mx-auto h-full">
-            <div className="absolute top-0 left-1/2 h-full border-l border-dashed border-cerulean path" />
+            <div className="absolute top-0 left-1/2 h-full border-l border-dashed border-cerulean last-of-type:h-0" />
           </div>
```

Comment:
![line](https://user-images.githubusercontent.com/4774314/151727923-abde04b2-aafb-4ee8-9680-0883343e0965.png)

can use this fixture to check -> https://github.com/TradeTrust/tradetrust-website/blob/master/src/test/fixture/ebl-endorsement-chain.json

Reply:
![image](https://user-images.githubusercontent.com/68947167/152291495-e543db6a-f964-4289-abaf-ef2a7682abcf.png)

can already, was referencing tailwind v3, that had those css classes